_playwright-tests/UI/Templates.spec.ts

```diff
@@ -8,11 +8,27 @@ test.describe('Templates', () => {
   }) => {
     await navigateToTemplates(page);
     await closePopupsIfExist(page);
-
-    const AddButton = page.locator('[data-ouia-component-id="create_content_template"]');
-
-    // Wait for the Add button to become enabled (up to 10 seconds)
-    await AddButton.first().isEnabled({ timeout: 10000 });
+    await page.getByRole('button', { name: 'Add content template' }).click();
+    await page.getByRole('button', { name: 'Select architecture' }).click();
+    await page.getByRole('option', { name: 'x86_64' }).click();
+    await page.getByRole('button', { name: 'Select version' }).click();
+    await page.getByRole('option', { name: 'el8' }).click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    // Add the step to select 'Red hat repos Checkbox'
+    await page.getByRole('button', { name: 'Next' }).click();
+    // Add the step to select 'checkbox for cutom repo'
+    await page.getByRole('button', { name: 'Next' }).click();
+    await page.locator('input[name="use-latest-snapshot"]').click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    const nameInput = page.getByPlaceholder('Enter name');
+    // const descriptionInput = page.getByPlaceholder('Enter Description');
+    await nameInput.fill('demo_template');
+    // await descriptionInput.fill('test');
+    await page.getByRole('button', { name: 'Next' }).click();
+    // Add steps to create teamplate with system if you want to
+    await page.locator('button.pf-v5-c-menu-toggle__button').click();
+    await page.getByRole('option', { name: 'Create template only' }).click();
+    await expect(page.getByText('Content Template "demo_template" created')).toBeVisible();
```

Comment:
Need help with this assertion

Reply:
> However, the template is not visible

Why is it not visible? The test should verify that the template is displayed and correct.